suseconnect-ng.spec

```diff
@@ -34,10 +34,13 @@ BuildRequires:  golang-packaging
 BuildRequires:  go >= 1.16
 BuildRequires:  zypper
 BuildRequires:  ruby-devel
+
 %if %{with hwinfo}
 %global test_hwinfo_args -test-hwinfo
+
+ExcludeArch: %ix86 s390 ppc64
```

Comment:
Just noticed this in the Factory submission review: this is only excluded when with hwinfo is enabled.

Was that actually intended?

Reply:
Oh, good catch, then we should move this out of the %if block.

Reply:
Moved the 'ExcludeArch' out of the 'if' block: https://github.com/SUSE/connect-ng/pull/163